server/src/utils.js

```diff
@@ -3,15 +3,15 @@
 import axios from "axios";
 
 export function getFileURL(file) {
-  const rawURL = `https://deliberation-assets.nyc3.cdn.digitaloceanspaces.com/${file}`;
+  const rawURL = `https://s3.amazonaws.com/assets.deliberation-lab.org/${file}`;
```

Comment:
just out of curiosity, why, does this need the s3.amazonaws.com prefix, rather than just using `https://assets.deliberation-lab.org`?

Reply:
It wasn't working for me and I couldn't really figure out why.  I think we would need to add all of the files as keys but I couldn't get it to work.  I can spend some more time looking into that if you'd like.